wurst/_handles/Effect.wurst

```diff
@@ -206,6 +208,30 @@ public function effect.clearSubAnimations()
 public function effect.removeSubAnimation(subanimtype whichSubAnim)
 	BlzSpecialEffectRemoveSubAnimation(this, whichSubAnim)
 
+
+// Extensions
+/** remove effect after "time" expires*/
+public function effect.destr(real time)
+	doAfter(time) ->	  
```

Comment:
I don't expect this to be inside handle's own API. ClosureTimers is an advanced thing in wurst that adds quite an overhead due to how lambda and method overriding work. Such functions fit much more in separate packages, but then I think that they're too simple to add to stdlib, especially if user will want to do something else aside from destroying it.

Reply:
Oh, this can actually reduce the amount of generated overhead because there's only one lambda for this sole purpose. Still, shouldn't be inside _handles/.